Dockerfile

```diff
@@ -31,4 +31,4 @@ RUN RAILS_ENV=production SECRET_KEY_BASE=`bin/rake secret` DB_ADAPTER=nulldb DAT
 
 FROM hyku-base as hyku-worker
 ENV MALLOC_ARENA_MAX=2
-CMD bundle exec sidekiq
+CMD ./bin/worker
```

Comment:
@orangewolf I believe this change was for LV and not Hyku proper. I got lost in the timeline when I saw the bundle exec sidekiq.

Reply:
you are correct

Reply:
@aprilrieger corrected